src/Fields/PageCombinedEventsListGroup.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * Copyright (c) 2023. Geniem Oy
+ */
+
+namespace TMS\Plugin\ManualEvents\Fields;
+
+use Geniem\ACF\Exception;
+use Geniem\ACF\Group;
+use Geniem\ACF\RuleGroup;
+use Geniem\ACF\Field;
+use TMS\Theme\Base\ACF\Field\TextEditor;
+use TMS\Theme\Base\ACF\Fields\EventsFields;
+use TMS\Theme\Base\Logger;
+
+/**
+ * Class PageCombinedEventsListGroup.
+ *
+ * @package TMS\Plugin\ManualEvents\Fields
+ */
+class PageCombinedEventsListGroup {
+
+    /**
+     * ManualEventGroup constructor.
+     */
+    public function __construct() {
+        add_action(
+            'init',
+            \Closure::fromCallable( [ $this, 'register_fields' ] )
+        );
+    }
+
+    /**
+     * Register fields
+     */
+    protected function register_fields() : void {
+        try {
+            $group_title = _x( 'Sivun asetukset', 'theme ACF', 'tms-plugin-manual-events' );
```

Comment:
Nämä voi mun mielestä olla näin ettei tu turhaa bloattia .po-tiedostoihin.
```suggestion
            $group_title = 'Sivun asetukset';
```